db/seeds.rb

```diff
@@ -5,3 +5,17 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+first_user = User.create(name: 'Harry', photo: 'https://images.pexels.com/photos/1851164/pexels-photo-1851164.jpeg', bio: 'Doctor from USA.')
+second_user = User.create(name: 'Noqs', photo: 'https://images.pexels.com/photos/325045/pexels-photo-325045.jpeg', bio: 'Doctor from USA.')
+third_user = User.create(name: 'John', photo: 'https://images.pexels.com/photos/45911/peacock-plumage-bird-peafowl-45911.jpeg', bio: 'Doctor from USA.')
+fourth_user = User.create(name: 'Jb', photo: 'https://images.pexels.com/photos/39317/chihuahua-dog-puppy-cute-39317.jpeg', bio: 'Doctor from USA.')
+fifth_user = User.create(name: 'Wang', photo: 'https://images.pexels.com/photos/847393/pexels-photo-847393.jpeg', bio: 'Doctor from USA.')
+sixth_user = User.create(name: 'Bob', photo: 'https://images.pexels.com/photos/617278/pexels-photo-617278.jpeg', bio: 'Doctor from USA.')
+seventh_user = User.create(name: 'Xi', photo: 'https://images.pexels.com/photos/1108099/pexels-photo-1108099.jpeg', bio: 'Doctor from USA.')
+
+first_post = Post.create(author: second_user, title: 'Hello', text: 'This is my first post')
+second_post = Post.create(author: first_user, title: 'Ola', text: 'I am the first user')
+
+Comment.create(post: first_post, author: first_user, text: 'Hi mary!' )
+Like.create(post: first_post, author: first_user )
```

Comment:
- [ ] I am sorry but you need to review your database migrations, your database does not reflect the content of your project. You need to set up very well your migrations. if you need help on this kindly contact me on slack @andersonlebon

Reply:
@andersonlebon I am trying to reach out to you on slack but the username you gave seem to be incorrect

Reply:
@Bludidi  Sorry for that, I miss spelled it, the right one is @Anderson